about.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+    ></script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  <link rel="stylesheet" href="about.css">
+  <title>cars show information</title>
+</head>
+<body>
+  
+  <header>
+    <aside class="top-section">
+      <div class="bar-container">
+      <ul class="bars">
+        <li>
+          <i class="fab fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>
+          <h4>English</h4>
+        </li>
+        <li>
+          <h4>My Page</h4>
+        </li>
+        <li>
+          <h4>Logout</h4>
+        </li>
+      </ul>
+      </div>
+    </aside>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="img/pakwheels.png" alt="plane" class="plane"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
```

Comment:
- [ ] Please give `position: fixed` to `mobile navbar` to prevent users from scrolling the screen. Currently it is possible to scroll the page with an open `navbar`: 
<img width="331" alt="Screenshot 2022-03-16 at 18 20 50" src="https://user-images.githubusercontent.com/77038610/158611266-4c4c0d26-b3f1-48a3-9f07-335e49b424be.png">